docs/coordinates/spectralcoord.rst

```diff
@@ -0,0 +1,530 @@
+    ...                         observer=alma, target=ttau)  # doctest: +IGNORE_WARNINGS +REMOTE_DATA
+
+We can convert these to the rest frame of the target using::
+
+    >>> sc_feat_rest = sc_feat.with_observer_velocity(sc_feat.target)  # doctest: +REMOTE_DATA
+    >>> sc_feat_rest  # doctest: +REMOTE_DATA
+    <SpectralCoord [115.27577801, 115.28177883, 115.28277896] GHz
+        observer:
+          <ICRS Coordinate: (x, y, z) in m
+              (-1.25867767e+11, -7.48979688e+10, -3.24757657e+10)
+           (v_x, v_y, v_z) in km / s
+              (9.34149908, 20.49579745, 7.99178839)>
+        target:
+          <ICRS Coordinate: (ra, dec, distance) in (deg, deg, pc)
+              (65.497625, 19.53511111, 144.321)
+           (pm_ra_cosdec, pm_dec, radial_velocity) in (mas / yr, mas / yr, km / s)
+              (1.37949782e-15, 1.46375638e-15, 23.9)>
+        observer to target (computed from above):
+          radial_velocity=0.0 km / s
+          redshift=0.0>
+
+The frequencies are very close to the rest frequency of the 12CO J=1-0 molecular line transition,
+which is 115.2712018 GHz. However, they are not exactly the same, so if the features we see are
+indeed from 12CO, then they are Doppler shifted compared to what we consider the rest frame of
+T Tau. We can convert these frequencies to velocities assuming the Doppler shift equation
+(in this case with the radio convention)::
+
+    >>> sc_feat_rest.to(u.km / u.s, doppler_convention='radio', doppler_rest=115.27120180 * u.GHz)  # doctest: +REMOTE_DATA
+    <SpectralCoord [-11.90160347, -27.50828539, -30.10939904] km / s
+        observer:
+          <ICRS Coordinate: (x, y, z) in m
+              (-1.25867767e+11, -7.48979688e+10, -3.24757657e+10)
+           (v_x, v_y, v_z) in km / s
+              (9.34149908, 20.49579745, 7.99178839)>
+        target:
+          <ICRS Coordinate: (ra, dec, distance) in (deg, deg, pc)
+              (65.497625, 19.53511111, 144.321)
+           (pm_ra_cosdec, pm_dec, radial_velocity) in (mas / yr, mas / yr, km / s)
+              (1.37949782e-15, 1.46375638e-15, 23.9)>
+        observer to target (computed from above):
+          radial_velocity=0.0 km / s
+          redshift=0.0
+    	doppler_rest=115.2712018 GHz
+    	doppler_convention=radio>
+
+Note that these resulting velocities are different from the ``radial_velocity``
+property (which is still zero here) - the latter is the difference in velocity
+between observer and target, while the former are how much the spectral values
+are Doppler shifted by relative to the rest frequency or wavelength.
+
+So if the features are indeed from 12CO, they have velocities of approximately -11.9, -27.5 and
+-30.1 km/s relative to the T tau rest frame.
+
+.. _spectralcoord-common-frames:
+
+Common velocity frames
+======================
+
+Any valid astropy coordinate frame can be passed to the
+:meth:`~astropy.coordinates.SpectralCoord.with_observer_velocity`
```

Comment:
So, this suggests one cannot actually just pass in a velocity (still haven't looked at actual code)...

Reply:
Correct - which was my objection to that name :) I agree the current name is still not right